neo4j/neo4j.go

```diff
@@ -56,8 +56,12 @@ func New(dbAddr string, size, timeout, retries int) (n *Neo4j, err error) {
 func (n *Neo4j) checkAttempts(err error, instanceID string, attempt int) error {
 	ctx := context.Background()
 	if !isTransientError(err) {
-		log.Event(ctx, "received an error from neo4j that cannot be retried", log.ERROR,
-			log.Data{"instance_id": instanceID, "error": err})
+		log.Error(
```

Comment:
@ChrysmOre - went with vertical stack. I'm still new to go so was suprised you have to put the terminal comma on line 63. This look ok to you?